test/system/comment_test.rb

```diff
@@ -61,4 +61,22 @@ class CommentTest < ApplicationSystemTestCase
     assert_selector("#{parentid} .comment .comment-body p", text: 'batman')
   end
  
+  test 'the text change on preview button click' do
```

Comment:
Hm, so did the original fix not resolve it, or is the test not passing but it should be?